packages/@uppy/transloadit/src/Assembly.js

```diff
@@ -53,70 +50,9 @@ class TransloaditAssembly extends Emitter {
   }
 
   connect () {
-    this.#connectSocket()
     this.#beginPolling()
```

Comment:
nit: if we only call `beginPolling`, we can inline the function in `connect`

Reply:
Yes, but to be honest I would even leave the code commented out, because we need to add SSE support there as soon as it lands in API2 @aduh95 @tim-kos